### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by BigFab**
This PR now includes the bump to 0.10.1

### CODE REVIEW

File: api/pin_reset.go [hunk 1]
```diff
@@ -46,10 +48,9 @@ func (a *Api) SendPinReset(res http.ResponseWriter, req *http.Request, vars map[
 	var usrDetails *shoreline.UserData
 	var err error
 
-	var token = req.Header.Get(TP_SESSION_TOKEN)
-
-	td := a.sl.CheckToken(token)
-	if td == nil || td.IsServer {
+	if td := a.token(res, req); td == nil {
```
Comment (franck-fourel) [inline api/pin_reset.go]:
```suggestion
	if token := a.token(res, req); token == nil {
```
to be consitent with the hydrophone code

File: api/pin_reset.go [hunk 2]
```diff
@@ -46,10 +48,9 @@ func (a *Api) SendPinReset(res http.ResponseWriter, req *http.Request, vars map[
...
+		return
+	} else if td.IsServer {
```
Comment (franck-fourel) [inline api/pin_reset.go]: Does it mean we consider we will never need to make pin reset on behalf of a user ?

File: templates/locales/en.yaml [hunk 3]
```diff
@@ -26,7 +26,7 @@ PatientPasswordResetHello: "Hello,"
 PatientPasswordResetRequest: "You requested a password reset."
 PatientPasswordResetDidNotRequest: "If you didn't request this, please ignore this email."
 PatientPasswordResetByDevice: "Resetting your password has to be done on your Diabeloop device."
-PatientPasswordResetExplainKey: "Follow the instructions with the below key on on your Diabeloop device."
+PatientPasswordResetExplainKey: "Follow the instructions with the below key on your Diabeloop device."
```
Comment (franck-fourel) [inline templates/locales/en.yaml]: not needed as https://github.com/mdblp/hydrophone/pull/92 will rework the patient password reset labels

**Review by franck-fourel — COMMENTED** [review]
couple of questions not necessarily important

File: templates/locales/en.yaml (hunk 3 above)
Comment (BigFab) [reply templates/locales/en.yaml]: We should leave it here anyway to prevent from repushing new changes

**Review by franck-fourel — APPROVED** [review]
LGTM

**Pull request merged**